assets/js/state/index.js

```diff
@@ -72,6 +74,19 @@ const processChannelEvents = (store) => {
     'database_instance_health_changed',
     'database_instance_system_replication_changed',
   ]);
+
+  // FIXME: This is to overcome the fact that we are generating names with registerEvents
+  // in the future we want to remove this and use the constants directly,
+  // since events and actions may have different names and parameters.
+  const channel = socket.channel('monitoring:executions', {});
+  channel.on('execution_completed', ({ group_id: groupID }) => {
+    store.dispatch({
```

Comment:
We should import and use the appropriate action here: 
https://github.com/trento-project/web/blob/main/assets/js/state/actions/lastExecutions.js#L3
```
store.dispatch(updateLastExecution(groupID))
```